src/modules/PublicLab.TitleModule.Related.js

```diff
@@ -26,72 +26,141 @@ Results should be in following JSON format:
 module.exports = function relatedNodes(module) {
 
   var relatedEl;
+  var addedRelatedEl;
+  var addedRelatedPost = 'addedTitles';
+  var relatedResultShow = false;
 
   build();
   bindEvents()
 
   // make an area for "related posts" to connect to
-  function build() { 
+  function build() {
 
     module.el.find('.ple-module-content').append('<div style="display:none;" class="ple-title-related"></div>');
     relatedEl = module.el.find('.ple-title-related');
-    relatedEl.append('<p class="ple-help">Does your work relate to one of these? Click to alert those contributors.</p><hr style="margin: 4px 0;" />');
+    relatedEl.append('<p class="ple-help">Does your work relate to one of these? Click to tag the related work.</p><hr style="margin: 4px 0;" />');
+    module.el.find('.ple-module-content').append('<div style="display:none;" class="ple-title-added"></div>');
+    addedRelatedEl = module.el.find('.ple-title-added');
 
   }
 
+  // showResult() will append the required tags in the particular container
+  function showResults(resultContainer, result, resultClass) {
+    resultContainer.find(resultClass + ' .title').html(result.title);
+    resultContainer.find(resultClass + ' .title').attr('href', result.url);
+    resultContainer.find(resultClass + ' .author').html('@' + result.author);
+    resultContainer.find(resultClass + ' .author').attr('href', '/profile/' + result.author);
+  }
+
+  function showAdded(addedResult, relatedResults) {
+    addedRelatedEl.append('<div class="addedresult addedresult-' + addedResult.id + '" style="margin: 3px;"><a class="btn btn-xs btn-default remove-tag"><i class="fa fa-times-circle"></i> Remove</a> <a class="title"></a> by <a class="author"></a></div>');
```

Comment:
Are the button outlines not appearing (despite `btn btn-xs btn-default) because the example files are on Bootstrap 3 and the `plots2` project is on Bootstrap 4? Is there a way to use CSS to achieve decent styling on both (like including `btn-link` or something) or should we just solve this in a new PR which bumps the Editor's Bootstrap to v4? I could have this wrong but just trying to troubleshoot.

https://getbootstrap.com/docs/4.4/components/buttons/

https://getbootstrap.com/docs/3.3/css/#buttons

Thank you!

Reply:
I will do it with using the css.